src/main/java/frc/robot/subsystems/manipulator/ArmSubsystem.java

```diff
@@ -286,6 +293,40 @@ private double getPositionFromDistance(double distance) {
     }
   }
 
+  /*
+   * Calculates the angle of the speaker to the arm pivot point plus the angle of the shooter.
+   *
+   */
+  public static double calculateAngleFromDistance(double distance) {
```

Comment:
This doesn't need to be made public or declared as static. We also never utilize it.

Reply:
It was static to make it easier to use in a test routine.   Agreed it doesn't need to be static, but also doesn't need to be non-static :) .